packages/fxa-auth-db-mysql/lib/db/schema/patch-114-115.sql

```diff
@@ -0,0 +1,54 @@
+--
+-- This migration introduces a table to track sent emails related to
+-- subscriptions.
+--
+
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('114');
+
+CREATE TABLE IF NOT EXISTS sentSubscriptionEmails (
```

Comment:
Probably be helpful to leave a comment here on what the different column store.

Reply:
Why not make this more generic so we can track any/all emails sent, not just subscription?

Reply:
I'm not against it.  Just didn't know people wanted/needed it.  I see below that you are considering a JSON column for arbitrary data.  Looks like the indexing can be a problem: https://dev.mysql.com/doc/refman/5.7/en/create-table-secondary-indexes.html#json-column-indirect-index